src/octopeer-github/content/ContentController.ts

```diff
@@ -105,7 +105,10 @@ class ContentController {
         this.unhookFromDOM();
         this.hookSemanticToDOM(database);
         this.hookTrackersToDOM(database);
-        this.hookMutationObserverToDOM();
+
+        if (Options.get("dataHTML")) {
```

Comment:
@MathiasMeuleman You probably want this: [436319b](https://github.com/thervh70/ContextProject_RDD/pull/168/commits/436319b11bc8db7d01bc5c155f0103b8448b2a5a)